src/model/UserphraseModel.h

```diff
@@ -71,4 +72,5 @@ public slots:
 
     std::unique_ptr<ChewingContext, void (*)(ChewingContext*)> ctx_;
     UserphraseSet userphrase_;
+    int addResult;
```

Comment:
>How do you think about my way to initialize the variable?

following the standard, it's ok.

I've notice another thing. `addResult` doesn't follow the coding style other private class member use: all lowercase and postfix a underscore.
This applies to other places, please check.